src/bundle.ts

```diff
@@ -242,11 +242,12 @@ function logWatcherBuildTime(result: RollupWatcher[]) {
 
   result.map((watcher) => {
     function start() {
-      if (startTime === 0) startTime = performance.now()
+      if (watcherCounter === 0) startTime = performance.now()
+      watcherCounter++
     }
     function end() {
-      watcherCounter++
-      if (watcherCounter === result.length) {
+      watcherCounter--
```

Comment:
Will this have concurrence issue? Last time I checked the number it's shorter than the actual build time

Reply:
How to reproduce the concurrence issue?

Reply:
You can log all the build duration and compare

Reply:
Take bunchee as an example, when I run the command `pnpm bunchee -w` in the project, it will start 4 jobs, watcherCounter will increase to 4 and then decrease to 0 one by one. After I modify `bundle.js`, it will restart 2 jobs, watcherCounter will increase to 2 and then decrease to 0.

Reply:
I was testing against multi-entries integration test, it has more jobs spinning up together. Is that one also logging the same data for you?

Reply:
I tried in multi-entries test, still feel good. 

Here are steps: 

1. open bunchee project, execute `pnpm build`, `pnpm install`, `npm link` in turn.
2. cd `test/integration/multi-entries`, execute `bunchee -w`
3. modify some files and saved.

Here is a sample:

```bash
bunchee -w
Watching project /Users/nnecec/Github/bunchee/test/integration/multi-entries...
0
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
21
20
19
18
17
16
15
14
13
12
11
10
9
8
7
6
5
4
3
2
1
0
✓ Build in 7681.08ms
0
1
1
0
✓ Build in 1155.92ms
0
1
1
0
✓ Build in 761.12ms
0
1
2
3
4
4
3
2
1
0
✓ Build in 2460.70ms
```

Reply:
sounds good, let's ship it, thanks!